idea/idea-fir/tests/org/jetbrains/kotlin/idea/quickfix/AbstractHighLevelIntentionTest.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.idea.quickfix
+
+import org.jetbrains.kotlin.idea.intentions.AbstractIntentionTest
+import org.jetbrains.kotlin.test.utils.IgnoreTests
+import java.io.File
+import java.nio.file.Paths
+
+abstract class AbstractHighLevelIntentionTest : AbstractIntentionTest() {
```

Comment:
there is already `org.jetbrains.kotlin.idea.fir.intentions.AbstractHLIntentionTest`

Reply:
Thanks! Let me use that!